docs/alerts/creating-logs-alerts.md

```diff
@@ -1,41 +1,30 @@
 title: Creating Logs Alerts
 description: Step-by-step alert creation instructions for Logs
 
-In a Logs App, *saved queries* are used to save search queries that you want to reuse.
+In a Logs App you can create an Alert Rule which will trigger notifications when a condition is met.
 
 For example, let's say you used *include/exclude filters* to search for *Elasticsearch slowlogs warnings*.
 
 ![Elasticsearch slowlogs warnings search](../images/alerts/image_0.png)
 
-Clicking on the bell icon creates a new *saved query*, where filters are transformed into `Query string`.
+Clicking on the bell icon creates a new Alert Rule with query and all filters applied.
 
 ![image alt text](../images/alerts/image_1.png)
 
-You can save this query as is and use it in future searches.
-
-![image alt text](../images/alerts/image_2.png)
-
-Clicking on the *alert view* icon <img role="presentation" width="26" height="26" src="/docs/images/alerts/image_27.png" style="display:inline"> opens the view where you can edit *saved queries*.
+Let's say we want to get notified if the number of *slowlog warnings* reaches 10 in any 10 minutes. Notice that *Chart Preview* shows the threshold line to help
+you visualize the threshold value in context.
```

Comment:
you visualize the threshold value in context.
=>
you visualize the threshold value in this context.